runtime/doc/pattern.txt

```diff
@@ -71,12 +71,14 @@ N			Repeat the latest "/" or "?" [count] times in
 			Only whole keywords are searched for, like with the
 			command "/\<keyword\>".  |exclusive|
 			'ignorecase' is used, 'smartcase' is not.
+			In Visual mode, search forward selection.
 
 							*#*
 #			Same as "*", but search backward.  The pound sign
 			(character 163) also works.  If the "#" key works as
 			backspace, try using "stty erase <BS>" before starting
 			Vim (<BS> is CTRL-H or a real backspace).
+			In Visual mode, search backward selection.
```

Comment:
We need to add help tags for the default values, as in:

https://github.com/neovim/neovim/blob/5d633546bf5990d03e4b4dc1df213f88316115e6/runtime/doc/insert.txt#L80-L82

So let's add tags for `v_*-default` and `v_#-default`, both of which should also have a link back to `*default-mappings*`.

Reply:
thx!
I tried to fix the documentation!
However, English may be strange because I'm not native.
If possible, could you please correct the wording and the test?